### DISCUSSION TITLE
Some ideas for additions

### DISCUSSION THREAD

**Discussion by dominiwe** [danielebruneo/remarkable2-hacks#11] [Ideas]
I recently found this: <https://github.com/timower/rM2-stuff>

There's not much documentation there but the author already made a [PR to add this stuff to toltec](https://github.com/toltec-dev/toltec/pull/261).

Also it would be cool if we had a reliable guide on how to compile/cross-compile for the reMarkable 2. There are docker images from toltec at <https://github.com/toltec-dev/toolchain> but sadly not a lot of documentation.
Any guide about cross-compilation from the reMarkable 1 era is outdated because [remarkable.engineering](https://remarkable.engineering) as well as their toolchain have been taken down.
That would definitely help accelerate the development of stuff for the rM2.

Another idea for an additional section would be a guide on how to install [`sshfs`](https://de.wikipedia.org/wiki/SSHFS) and mount the filesystem of the reMarkable. I've also been messing around with `rclone` and mounting cloud storage on the reMarkable directly.

Great repo by the way, really helped me get started :)

**Comment by danielebruneo** [comment]
Thanks for you valuable comment :)
Lot of stuff, all very interesting!

I look forward to try that quick terminal emulator, cause fingerprint is really quite unusable on the device.

I would like to write something for the RM2 so, I will eventually go through the cross compiling process and will document that. 
Also if I'd probably rather chose some interpreted env like python over binaries for a non critical tool (performance should not be an issue here, as a developer I suspect your main limit will be screen refresh time rather than cpu time).
But definitively there's need for multiple guides on how to approach development for RM2.

Really good idea the guide on `sshfs`, that's part of my usual flow but I see many (and many experienced) users do not know or think about it.


Wondering if you maybe interested in collaborate to the repo @DezMakes ?
There's lot of work to do and we need to be an actual comunity starting somewhere... :)

**Reply by dominiwe** [reply]
> I look forward to try that quick terminal emulator, cause fingerprint is really quite unusable on the device.

I tried it out, it's really fast because it draws to the framebuffer directly. Much needed and hopefully in toltec testing soon. 

> Wondering if you maybe interested in collaborate to the repo @DezMakes ?
> There's lot of work to do and we need to be an actual comunity starting somewhere... :)

Yeah, would love to help out. :)
Just got my reMarkable a couple days ago, so I'm exploring everything!
